python/ray/autoscaler/_private/autoscaler.py

```diff
@@ -716,3 +743,105 @@ def kill_workers(self):
             self.provider.terminate_nodes(nodes)
         logger.error("StandardAutoscaler: terminated {} node(s)".format(
             len(nodes)))
+
+
+def format_info_string(lm_summary, autoscaler_summary):
+    header = "=" * 8 + f"Autoscaler status: {datetime.now()}" + "=" * 8
+    available_node_report_lines = []
+    for node_type, count in autoscaler_summary.active_nodes.items():
+        line = f"\t{count} {node_type}"
```

Comment:
I really liked @ericl's original format (specifically regarding the whitespace usage).

Instead of `\t` can we use single or double space?